build.gradle

```diff
@@ -333,7 +333,7 @@ configurations.all {
 }
 
 dependencies {
-  implementation 'com.github.hmcts:civil-commons:1.0.12'
+  implementation 'com.github.hmcts:civil-commons:1.0.12-civ-9123-9407'
```

Comment:
This is a temporary change for testing purposes. A full release with the required functionality should be added once available.